src/core/reactor.cc

```diff
@@ -2649,30 +2649,31 @@ void reactor::register_metrics() {
             sm::make_counter("abandoned_failed_futures", _abandoned_failed_futures, sm::description("Total number of abandoned failed futures, futures destroyed while still containing an exception")),
     });
 
-    using namespace seastar::metrics;
+
```

Comment:
might want to remove this empty line.

also

> makes the metrics registration generic

could be confusing, as generic could mean different things in different contexts. could you use something like 

> qualify metric function calls with "sm::"

?

Reply:
forced push v2 with the changes